tests/ltp/batch.mk

```diff
@@ -22,7 +26,10 @@ ESCALATE_CMD=sudo
 $(ALPINE_TAR):
 	curl -L -o "$@" "https://nl.alpinelinux.org/alpine/v$(ALPINE_MAJOR)/releases/$(ALPINE_ARCH)/alpine-minirootfs-$(ALPINE_VERSION)-$(ALPINE_ARCH).tar.gz"
 
-$(ROOT_FS): $(ALPINE_TAR) ../buildenv.sh
+$(LTP_TEST_MNT_IMG):
+	dd if=/dev/zero of=$(LTP_TEST_MNT_IMG) count=$(LTP_TEST_MNT_IMG_SIZE) bs=1M
+
```

Comment:
Why this doesn't have line
mkfs -t ext4 $(LTP_TEST_MNT_IMG)

Reply:
done